app/views/guests/show.html.erb

```diff
@@ -0,0 +1,19 @@
+<h1> <%= @guest.name %> </h1>
+
+<div class = "guest_rooms">
+  <% @guest.rooms.each do |room| %>
+    <div id = <%="#{room.id}"%>>
+      <h3>Suite: <%= room.suite %></h3>
+      <p>Nightly Rate: <%= room.rate %>$</p><br>
+      <p>Hotel: <%= room.hotel.name %></p><br>
+    </div>
+  <% end %>
+</div>
+
+<div class = "add_room_form">
+  <%= form_with url: "/guest_rooms/#{@guest.id}", method: :post, local: true do |form| %>
```

Comment:
Excellent path/verb combo here, it's exactly what I would have done 🙌

Reply:
An alternative that might be easier to read would be
```
url: "/guests/:guest_id/rooms"
```
and then send in the room Id as you have done in the form.